modules/transactionpool/persist.go

```diff
@@ -53,6 +55,9 @@ func (tp *TransactionPool) initPersist() error {
 	}
 
 	// Open the database file.
+	if build.DEBUG && build.Release != "testing" {
+		fmt.Println("\tTransaction Pool: opening database")
+	}
```

Comment:
No it's meant to stay there for the time being. The transcation pool occasionally hangs at startup (indefinitely) and I'm trying to figure out where in the code it gets stuck.